oxidation/client/tests/unit/common/storage.spec.ts

```diff
@@ -47,8 +48,8 @@ describe('Storage manager', () => {
     await manager.create();
 
     const d: {[slug: string]: StoredDeviceData} = {
-      'abcd': {lastLogin: new Date('2022-02-02T00:00:00.000Z')},
-      'efgh': {lastLogin: new Date('2011-01-01T00:00:00.000Z')}
+      'abcd': {lastLogin: DateTime.fromISO('2022-02-02T00:00:00.000+00:00', {setZone: true})},
```

Comment:
why the need to switch from 'Z' to '+00:00'?

Reply:
Just tried a lot of different formats until I found out about the `setZone: true`. But it shouldn't make a difference. I do like the `+00:00` better though, it makes it a little more explicit.

Reply:
LGTM